src/framework/script/script-registry.js

```diff
@@ -20,6 +20,12 @@ class ScriptRegistry extends EventHandler {
      */
     _list = [];
 
+    /**
+     * @type {Map<string, import('./script-attributes.js').AttributeSchema} - A Map of script names to attribute schemas
```

Comment:
The description goes above the `@type` tag. And our convention is to leave a blank link between the description and the first tag line.